packages/flutter_tools/test/general.shard/android/android_gradle_builder_test.dart

```diff
@@ -885,113 +885,13 @@ Gradle Crashed
         command: <String>[
           'gradlew',
           '-q',
-          'printFreeDebugApplicationId',
+          'dumpFreeDebugAppLinkSettings',
```

Comment:
Shouldnt this have a expect to make sure your code works?

Reply:
Test will fail if it doesn't send out command declared in FakeCommand